lib/codit/population/networks/city.py

```diff
@@ -88,13 +100,17 @@ def build_households(people):
     n_individuals = len(people)
     assigned = 0
     households = []
-
     num_h = int(n_individuals / AVERAGE_HOUSEHOLD_SIZE)
     household_examples = build_characteristic_households(num_h)
+    # create num_h of homes
+    homes_list = build_households_home_list(num_h)
```

Comment:
could this variable be called `homes_examples`? This would put it on the same level as `household_examples`. Perhaps it could even be `accommodation_examples`?

Or have I not understood its role and purpose?

Reply:
homes_examples is OK. Though the coordinates of these homes are real, I use the full list of homes as weighted examples. So I think homes_examples is also accurate.